ansible_collections/arista/avd/roles/eos_l3ls_evpn/tasks/main.yml

```diff
@@ -13,7 +13,14 @@
   delegate_to: localhost
   tags: [build, provision]
 
+- name: Create Directory for the dc topology documentaion
```

Comment:
This is not required, we have a dedicated role to build all output folders: https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/roles/build_output_folders

The reason for this approach is that it is only required once.

Reply:
Well we need to create folders per fabric_name to support multiple PODs per DC. But I have moved logic to the build_output_folders.